lib/http/2/stream.rb

```diff
@@ -173,16 +173,15 @@ def reprioritize(weight: 16, dependency: 0, exclusive: false)
     # @param payload [String]
     # @param end_stream [Boolean] indicates last response DATA frame
     def data(payload, end_stream: true)
-      flags = []
-
       # Split data according to each frame is smaller enough
       # TODO: consider padding?
       max_size = @connection.remote_settings[:settings_max_frame_size]
       while payload.bytesize > max_size
         chunk = payload.slice!(0, max_size)
-        send(type: :data, flags: flags, payload: chunk)
+        send(type: :data, flags: [], payload: chunk)
```

Comment:
Doh, right, should have caught that in the previous pull. 

DATA frames can also be padded, so if we're splitting a large payload we should probably pass the flag through (if specified) and pad each frame. That said, we don't actually support padding today, so that's a bit of a noop. I guess we  already have the TODO above, so it's ok as is.